.github/workflows/ci.yml

```
@@ -6,27 +6,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.4', '8.0', '8.1', '8.2']
+        #php-versions: ['8.1', '8.2', '8.3', '8.4']
+        php-versions: ['8.4']
```

Need to add back currently supported versions

```suggestion
        php-versions: ['8.1', '8.2', '8.3', '8.4']
```